src/main/webapp/app/entities/result/result-detail.component.html

```diff
@@ -7,10 +7,13 @@ <h4 class="modal-title">Feedback</h4>
         <fa-icon [spin]="true" size="lg" icon="circle-notch"></fa-icon>
     </div>
     <div [hidden]="isLoading" *ngFor="let feedback of feedbackList | removepositiveautomaticfeedback; last as isLast">
-        <p *ngIf="feedback.type === 'AUTOMATIC' && !feedback.positive">
+        <p *ngIf="feedback.type === 'AUTOMATIC' && feedback.positive === false">
```

Comment:
There is now a third state for feedbacks: the case that there is undefined feedback (neither positive nor negative, the case that e.g. in a programming-exercise a test was not executed).